setup.py

```diff
@@ -346,6 +347,17 @@ def get_extensions():
 
     ffmpeg_exe = distutils.spawn.find_executable('ffmpeg')
     has_ffmpeg = ffmpeg_exe is not None
+    if has_ffmpeg:
+        try:
+            ffmpeg_version = os.popen('ffmpeg -version | head -n1').read()
```

Comment:
Nit: I think we should close the process by calling `close()`. Also note that typically using the `subprocess` is recommended instead of `os.popen`.

Reply:
Changed it to use `subprocess.run()`.